frontend/src/hooks/api/actions/useEnvironmentApi/useEnvironmentApi.ts

```diff
@@ -82,6 +83,26 @@ const useEnvironmentApi = () => {
         }
     };
 
+    const cloneEnvironment = async (
+        name: string,
+        payload: IEnvironmentClonePayload
+    ) => {
+        const path = `api/admin/environments/${name}/clone`;
+        const req = createRequest(
+            path,
+            { method: 'POST', body: JSON.stringify(payload) },
+            'cloneEnvironment'
+        );
+
+        try {
```

Comment:
I don't think this is a big deal but does this try/catch do anything? Looks like the kind of try catch I put to trap a message with a debugger in development. Can this not be reduced to just

`return await makeRequest(req.caller, req.id);`

Reply:
You're absolutely correct, and any exceptions are getting caught by the `handleSubmit` anyways. It seems like this is consistent though, since it seems like other API hook methods generally look like this, but I'm not sure on the original reasoning there. I'll at least change this one to be more straightforward.